oak/proto/oak_api.proto

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak;
+
+// Status values exchanged as i32 values across the Node Wasm interface.
+enum OakStatus {
+  // Success.
+  OK = 0;
```

Comment:
I would suggest following best practices and having a safer default value, e.g. `OAK_STATUS_UNSPECIFIED` (see https://cloud.google.com/apis/design/naming_convention#enum_names). The enum below kind of does, but it could also follow the same naming convention for consistency.

Reply:
Done.